include/alpaka/core/RuntimeMacros.hpp

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2022 Andrea Bocci, Mehmet Yusufoglu */
+#pragma once
```

Comment:
please add an empty line above and after the pragma

Reply:
added, thanks.